src/js/components/Menu.js

```diff
@@ -149,6 +160,13 @@ var MenuDrop = React.createClass({
     if (this.props.small) {
       classes.push(CLASS_ROOT + "__drop--small");
     }
+    if (this.props.className) {
+      var classNames = this.props.className.split(" ");
+      classNames = classNames.map(function(className) {
+        return className + "__drop";
```

Comment:
We currently have a case with the HPE header and footer where one has a menu on the right side, and one has a menu on the left.  They need to be styled differently (at least so the `...` stays in the correct position for each menu), but that can't happen without the specific classes.

For example, in the footer, the `...` icon is on the right side. However, the `...` shows up on the left in the drop down menu instead.

![screen shot 2015-12-08 at 11 43 48 pm](https://cloud.githubusercontent.com/assets/7669/11679433/e93af51a-9e05-11e5-8b37-d27472fc5bfa.png)

![screen shot 2015-12-08 at 11 43 26 pm](https://cloud.githubusercontent.com/assets/7669/11679436/ecbc03a0-9e05-11e5-93a2-b4b17156fc92.png)